---
layout: post
title: Fri. Jul. 12, 2019  Salmon-Sea lice Zymo Pico Methyl prep QC and sequencing
---

## Zymo Pico Methyl kit prep QC

### Bioanalyzer

I used the Chip Priming station, Agilent chip votex, and Bioanalyzer 2100 in the Seebs lab.

**PROCEDURE:**
- I ran the Bioanalyzer 2100 high sensistivity DNA assay on all the samples. To do this, I followed the [quick start guide](https://github.com/RobertsLab/resources/blob/master/protocols/Commercial_Protocols/Agilent_Bioanalyzer_HighSensitivityDNAKit_QuickStartGuide.pdf).
	- I first rinsed the electrodes with an electrode cleaning chip following protocol on [page 118 of the maintenance manual](https://github.com/RobertsLab/resources/blob/master/equipment_manuals/Agilent%202100-Bioanalyzer_Maintenance-Troubleshooting_USR_ENG.pdf)
	- I did not adjust the base plate of the chip priming station since it was already set at C and it did not appear that anyone had done that before.
	- I wasted my first chip because I did not prime it correctly:
		- I got the error message listed on [page 73 of the troubleshooting manual](https://github.com/RobertsLab/resources/blob/master/equipment_manuals/Agilent%202100-Bioanalyzer_Maintenance-Troubleshooting_USR_ENG.pdf) so I followed the directions on page 139 to perfom the seal test in the priming station and I realized the lock hadn't latched with an audible click when I first did it!
		- This was for the better anyways because when I vortexed this chip at 2400rpm as the quick guide specifies, I thought I saw a little liquid spraying so I turned it down to 2200rpm **NOTE FOR FUTURE: 2200rpm is enough**
	- It took about 6 minutes for the software to start showing the traces from the ladder, then each subsequent sample took about 3 minutes. There were a total of 11 samples (samples 1-11) on one chip.
	- Right before running the next chip with samples 12-20
		- rinsed the electrodes again with the same cleaning chip following [page 118 of the maintenance manual](https://github.com/RobertsLab/resources/blob/master/equipment_manuals/Agilent%202100-Bioanalyzer_Maintenance-Troubleshooting_USR_ENG.pdf)
		- the software froze and would only reopen in demo mode so I restarted the laptop, reopened the program, and it finally behaved. This took about 10 minutes, and the manual says to use the chip within 5 minutes. But it still ran fine judging by the ladder and how the samples looked similar to the first run.

**RESULTS:**

- [Report for Samples 1-11](https://drive.google.com/file/d/1oKAD7VP2GQgVtK6GFhDsZsZNOSduWo_7/view?usp=sharing)

- [Report for Samples 12-22](https://drive.google.com/file/d/1pPr8MAqX0NeieDy2Mo-K2qSqIIgrSU7U/view?usp=sharing)

- [raw data here](https://drive.google.com/drive/folders/1WZ3GJKF61kSzA8kzM4rhZjGu9Jf10WiC?usp=sharing)

**CONCLUSIONS:**

- Similarity between lanes 20 and 22 confirms samples 21 and 20 labels did get swapped as previously suspected. So sample #20 is Sea lice Female 1.

- Gel images generally look in the size range of the example in the [zymo pico methylseq manual](https://github.com/RobertsLab/resources/blob/master/protocols/Commercial_Protocols/ZymoResearch_PicoMethylseq.pdf) so that's encouraging.

![](https://drive.google.com/drive/folders/1gxsIXLgfIpCrzpmxtee0HkxnPnx_QRAa?usp=sharing)


### Qubit
I measured concentrations with Qubit HS DNA assay and recorded concentrations [here](https://docs.google.com/spreadsheets/d/1fkfBJJGvQqt009q3URor5AfrQZgTNGJpnDjcLoC6js0/edit?usp=sharing) under the 'sequencing' tab.

**CONCLUSIONS:**

- The yields ranged from 25-134ng, which suggest the library prep worked, considering I only started with between 20-50ng of DNA and went through all those steps.

- Bioanalyzer and Qubit concentrations generally agree, but in the past I've relied on Qubit and gel to determine the amount of DNA in the library, so I used those to calculate the nanomolar concentrations.

### Pooling samples
- I calculated nanomolar concentrations using the calculator under the tools menu in the application [EnzymeX](https://nucleobytes.com/enzymex/index.html)
	- you can enter in the average size(bp) of your library and 10nM (it says pmol, but it's the same for nM if you consider the weight to be ng/uL instead of ug). This is the protocol followed by the Ecker lab.
	- Example: for a 240bp library, a 1.584ng/uL concentration is a 10nM concentration.
![](https://drive.google.com/file/d/12SFR1GXq4qyD_4SOeazUrKeb-kmIuE9z/view?usp=sharing)

- I entered these concentrations into [this spreadsheet](https://docs.google.com/spreadsheets/d/1lkEmFfEFpfAWEZwB-t475lKkZEEk-RpGUPDiDfGZI3Y/edit?usp=sharing) under the column 'Conc_for_10nM(based on Qubit and gel avg)'.
- I calculated dilutions volumes in the [same spreadsheet](https://docs.google.com/spreadsheets/d/1lkEmFfEFpfAWEZwB-t475lKkZEEk-RpGUPDiDfGZI3Y/edit?usp=sharing) under columns 'vol_dna_for_10nM' and 'vol_water_for_10nM' and prepared these dilutions to get every sample at 10nM.
- Because I wanted each salmon sample to get 3.5% of reads and each sea lice sample to get 15% of reads, I made a 30uL pool by adding 1.05uL of each salmon sample and 4.5uL of each sea lice sample.

### Submitting samples to NW Genomics Core
- I submitted a signed quote for a NovaSeq SP 300 cycle flowcell (to get 1.6B 150bp PE reads) and a filled out [library submission form](https://github.com/RobertsLab/resources/blob/master/UW_NWGC_Library_Submission_Form_2019.docx)
- I dropped off my 30uL sample pool with Dolores at NWGC (ground floor of Genome Sciences).
	- She said the kit is coming next Tuesday and she will try to start it next week if the machine is available
	- She is going to load my library at 270pM
	- It only takes 1.5 days to run so she's hoping to send the data by the end of the week after next.

**FINGERS CROSSED!!!**
